docs/modules/build-system/cli.md

```diff
@@ -19,10 +20,30 @@ Unity.exe -projectPath "${GDK_PROJECT_PATH}" \
     -logfile ${LOG_FILE_PATH} \
     -executeMethod "Improbable.Gdk.BuildSystem.WorkerBuilder.Build" \
     +buildWorkerTypes "UnityClient" \
-    +buildTarget "cloud" \
+    +buildEnvironment "cloud" \
     +scriptingBackend "mono"
 ```
 
 This command builds the `UnityClient` worker type for cloud with the `mono` scripting backend.
 
-> **Note:** `Unity.exe` may not be in your path. You may need to call it via a fully qualified path.
\ No newline at end of file
+> **Note:** `Unity.exe` may not be in your path. You may need to call it via a fully qualified path.
+
+### Filtering build targets
+
+To build workers for a specific build target of a build environment, you need to provide the `buildTargetFilter` argument like the following:
+
+```bash
+Unity.exe -projectPath "${GDK_PROJECT_PATH}" \
+    -batchmode \
+    -quit \
+    -logfile ${LOG_FILE_PATH} \
+    -executeMethod "Improbable.Gdk.BuildSystem.WorkerBuilder.Build" \
+    +buildWorkerTypes "UnityClient" \
+    +buildEnvironment "cloud" \
+    +scriptingBackend "mono" \
+    +buildTargetFilter "macos"
+```
+
+This command builds the `UnityClient` worker type for cloud with the `mono` scripting backend, but only for MacOS. A `UnityClient` for Windows is _not_ built.
+
+> **Note:** The build target filter must be a subset of the available build targets for a given worker type on a given environment.
```

Comment:
```suggestion
> **Note:** The build target filter must be a subset of the available build targets for a given worker type on a given environment. Otherwise, an error will be thrown.
```

I think?

Reply:
yup - throws a nice BuildFailedException with the offending target